### ISSUE TITLE
Global flag for `up env`

### ISSUE THREAD

**Issue by tj** [apex/up#535]
Currently they are all scoped to an app, but it may be useful to have `--global`, this could act as a poor-mans "service discovery" if you don't want to bother with DNS (or another solution). The downside of course is that they'd be ... global haha.

**Comment by ngallardo** [comment]
So would these just be manageable from any Up deployed application directory?  It's not a bad idea but, like you said...  then they're all global.

**Comment by tj** [comment]
Yep they'd be account-wide for use between applications. I have mixed feelings about it but yep haha

**Comment by tj** [comment]
Actually, having second thoughts about this now because it would require two calls to SSM, or many more if the prefix is less specific. If "baking" in the variables (#547) happened then it would be viable.
